Library/ViewModels/CommentsViewModel.swift

```diff
@@ -86,8 +86,10 @@ public final class CommentsViewModel: CommentsViewModelType,
       }
 
     self.configureCommentComposerViewWithData = Signal
-      .combineLatest(initialProject.signal, currentUser.signal)
-      .takeWhen(self.viewDidLoadProperty.signal)
+      .combineLatest(initialProject, currentUser.signal, self.viewDidLoadProperty.signal.ignoreValues())
```

Comment:
This doesn't work because it's waiting for the project to load in order to configure the composer view and the view is temporarily shown in a broken state until configured:

<img width="50%" alt="image" src="https://user-images.githubusercontent.com/3735375/121605025-7d8aef00-ca00-11eb-8079-0bab80663ea8.png">

We either need to update `CommentComposerView` to improve how it looks until configured or manage this a different way. Possibly hide the composer until configured? cc @tobitech.

Reply:
Ok, I saw your comment now:

> Ignore the rough transition that happens on the composer, it's going to be fixed in this ticket https://kickstarter.atlassian.net/browse/NT-2009

That's fine.